Assignment2/Josesilva99-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,52 @@
+{
+
+"@context": {
+		"base": "http://example.org/assigment2/",
+		"includes": "Class01:includes",
+		"hasMeasurement": "Sensor029:hasMeasurement",
+		"hasTemperature": "Measurement8401:hasTemperature",
+		"hasOwner": "Computer101:hasOwner",
+		"hasName": "User10A:hasName",
+		"atTime":"Measurement8401:atTime",

```

Comment:
Change the prefix for your attribute definitions since you remove Class01, Sensor029, etc. You can reuse your base: instead.
Also, check the type definitions below to redefine them with your base: prefix.